json/Webhooks-v1.json

```diff
@@ -4042,6 +4176,315 @@
             ],
             "type" : "object"
          },
+         "DisputesNotificationAdditionalData" : {
```

Comment:
if feasible, would be good to include the issuerComments.* as seen here 


<img width="807" height="550" alt="image" src="https://github.com/user-attachments/assets/ac73c4d7-2c1b-4655-a21d-2943a79fda68" />

Reply:
Yes, good point

Reply:
This has been added